    - Make a few include files be self-skipping and self-complete.

 -- Matthias Urlichs <smurf@debian.org>  Tue, 15 Mar 2005 11:09:52 +0100

Index: ntp-4.2.4p0+dfsg/include/ntp_stdlib.h
===================================================================
--- ntp-4.2.4p0+dfsg.orig/include/ntp_stdlib.h	2006-12-28 13:03:05.000000000 +0100
+++ ntp-4.2.4p0+dfsg/include/ntp_stdlib.h	2007-04-10 10:47:13.000000000 +0200
@@ -4,6 +4,7 @@
 #include <sys/types.h>
 #include <sys/socket.h>
 #include "ntp_rfc2553.h"
+#include "ntp_refclock.h"
 
 #include "ntp_types.h"
 #include "ntp_string.h"
Index: ntp-4.2.4p0+dfsg/include/ntp_control.h
===================================================================
--- ntp-4.2.4p0+dfsg.orig/include/ntp_control.h	2006-06-06 22:16:17.000000000 +0200
+++ ntp-4.2.4p0+dfsg/include/ntp_control.h	2007-04-10 10:47:13.000000000 +0200
@@ -1,3 +1,5 @@
+#ifndef NTP_CONTROL
+#define NTP_CONTROL
 /*
  * ntp_control.h - definitions related to NTP mode 6 control messages
  */
@@ -285,3 +287,5 @@
 #define	TYPE_SYS	1
 #define	TYPE_PEER	2
 #define	TYPE_CLOCK	3
+
+#endif /* NTP_CONTROL */
Index: ntp-4.2.4p0+dfsg/include/ntpd.h
===================================================================
--- ntp-4.2.4p0+dfsg.orig/include/ntpd.h	2006-12-28 13:03:06.000000000 +0100
+++ ntp-4.2.4p0+dfsg/include/ntpd.h	2007-04-10 10:47:13.000000000 +0200
@@ -9,6 +9,7 @@
 #include "ntp_select.h"
 #include "ntp_malloc.h"
 #include "ntp_refclock.h"
+#include "ntp_control.h"
 #include "recvbuff.h"
 
 
Index: ntp-4.2.4p0+dfsg/include/ntp.h
===================================================================
--- ntp-4.2.4p0+dfsg.orig/include/ntp.h	2006-12-28 13:03:04.000000000 +0100
+++ ntp-4.2.4p0+dfsg/include/ntp.h	2007-04-10 10:47:13.000000000 +0200
@@ -5,6 +5,7 @@
 #define NTP_H
 
 #include "ntp_types.h"
+#include "ntp_fp.h"
 #include <math.h>
 #ifdef OPENSSL
 #include "ntp_crypto.h"
